helper/identity/types.proto

```diff
@@ -108,8 +119,8 @@ message Alias {
 	// ID is the unique identifier that represents this alias
 	string id = 1;
 
-	// EntityID is the entity identifier to which this alias belongs to
-	string entity_id = 2;
+	// ParentID is the entity identifier to which this alias belongs to
+	string parent_id = 2;
```

Comment:
I don't particularly like this change; if the point is that an alias is just another name for the same thing, it's not really a parent. It's just the associated entity.

Reply:
We had talked about doing this change particularly to make Alias struct to be used for both Entities and Groups, and the reasoning was that the object it belongs to (entity or the group) can be generically referred as a parent.

Reply:
Hm, that might have been an Armon comment. I wonder if we can figure out something better. There is no actual parent/child relationship; where we *do* have a parent/child relationship we instead chose to call it something else (member groups instead of child groups).

Maybe something like CanonicalID? That would fit pretty well since the alias is referring to an authoritative ID. Poll the room?

Reply:
Discussed offline and decided on CanonicalID. Changes pushed.